var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -38,13 +38,14 @@ class Conduit(CMakePackage):
     git = "https://github.com/LLNL/conduit.git"
     tags = ["radiuss", "e4s"]
 
-    license("Apache-2.0")
+    license("BSD-3-Clause")
```

Comment:
FWIW. If the licensing changed you can have multiple `license` directives that include a `when` argument. Examples are in the docs https://spack.readthedocs.io/en/latest/packaging_guide.html#specifying-license-information.

Reply:
@tldahlgren  thanks - but the license didn't change,  it just wasn't listed correctly in the spack package.